Sources/Views/InputTextView.swift

```diff
@@ -49,6 +49,13 @@ open class InputTextView: UITextView {
         }
     }
     
+    /// The images that are currently stored as NSTextAttachment's
+    open var images: [UIImage] {
+        return getAttachedImages()
+    }
+    
+    open var isImagePasteEnabled: Bool = true
```

Comment:
Maybe we can add a property in `MessageInputBar`? , I think put ` isImagePasteEnabled` in `inputTextView` maybe a little bit deep.

Reply:
Fair point, but I think it should still be apart of `InputTextView`. Maybe add the following in `MessageInputBar` ?

```
open var isImagePasteEnabled: Bool {
   get {
       return inputTextView. isImagePasteEnabled
   }
   set {
      inputTextView.isImagePasteEnabled = newValue
   }
}
```

Reply:
:+1: That's what I think.

Reply:
@nathantannar4 ,Add `isImagePasteEnabled` property in `MessageInputBar` is more friendly, and we should set `isImagePasteEnabled` property's access level to `internal` in `inputTextView` to prevent from inconsistently.

Reply:
What do you think?